styles/styles.css

```diff
@@ -0,0 +1,270 @@
+* {
+  box-sizing: border-box;
+}
+button, input, select {
+  font-family: inherit;
+  font-size: 100%;
+}
+header {
+  background-image: url(../images/header-image.jpg);
+  background-size: cover;
+}
+footer {
+  color: white;
+  background-color: rgb(13, 45, 78);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 3rem 0 3rem;
+}
+body {
+  margin: auto;
+  font-size: 16px;
+}
+main {
+  max-width: 1440px;
+}
+.logo {
+  padding: 3rem 0rem 0rem 6rem;
+  width: 100%;
+  min-width: 320px;
+}
+.tagline {
+  font-size: 1.55rem;
+  margin-top: -1rem;
+  padding: 0rem 0rem 10rem 6rem;
+  color: rgb(116, 171, 42);
+}
+.cost {
+  color: white;
+}
+.checkout {
+  color: white;
+  font-weight: bold;
+  background-color: rgb(116, 171, 42);
+  padding: .01rem 2rem;
+  border-radius: .1rem;
```

Comment:
I do like pixels for this, but that's just my opinion.

also....0.1 rem isn't that much. :)